src/css/flags/audioplayer.less

```diff
@@ -1,5 +1,28 @@
-.jwplayer {
-    &.jw-flag-audio-player {
+.jwplayer.jw-flag-audio-player {
+    background-color: transparent;
+
+    .jw-preview {
         background-color: transparent;
     }
+    .jw-controlbar {
+        display: table;
```

Comment:
We should have this one from imports/controlbar.less, are there others that we can also prune out?